openslides_backend/action/actions/meeting/clone.py

```diff
@@ -99,6 +102,13 @@ def duplicate_mediafiles(self, json_data: Dict[str, Any]) -> None:
                     mediafile["id"], self.replace_map["mediafile"][mediafile["id"]]
                 )
 
+    def add_meeting_title_suffix(self, instance: Dict[str, Any]) -> None:
+        meeting = self.get_meeting_from_json(instance["meeting"])
+        filter_ = FilterOperator("committee_id", "=", meeting["committee_id"])
+        counts = self.datastore.count(Collection("meeting"), filter_)
+        new_name = (meeting.get("name") or "") + f" ({counts + 1})"
```

Comment:
```suggestion
        new_name = meeting.get("name", "") + f" ({counts + 1})"
```
nit-pick

Reply:
It was precaution. I changed it to `meeting.get("name", "")`.